tests/integration/test_utils/data_source_config/snowflake.py

```diff
@@ -0,0 +1,133 @@
+from random import randint
+from typing import TYPE_CHECKING, Dict, List, TypeVar, Union
+
+import pandas as pd
+import pytest
+
+from great_expectations.compatibility.pydantic import BaseSettings
+from great_expectations.compatibility.sqlalchemy import (
+    Column,
+    MetaData,
+    Table,
+    create_engine,
+    insert,
+)
+from great_expectations.compatibility.typing_extensions import override
+from great_expectations.datasource.fluent.interfaces import Batch
+from tests.integration.test_utils.data_source_config.base import (
+    BatchTestSetup,
+    DataSourceTestConfig,
+)
+
+if TYPE_CHECKING:
+    from great_expectations.compatibility.snowflake import SnowflakeType
+
+_SnowflakeType = TypeVar("_SnowflakeType")
+
+
+class SnowflakeDatasourceTestConfig(DataSourceTestConfig):
+    @property
+    @override
+    def label(self) -> str:
+        return "snowflake"
+
+    @property
+    @override
+    def pytest_mark(self) -> pytest.MarkDecorator:
+        return pytest.mark.snowflake
+
+    @override
+    def create_batch_setup(
+        self, data: pd.DataFrame, request: pytest.FixtureRequest
+    ) -> BatchTestSetup:
+        return SnowflakeBatchTestSetup(
+            data=data,
+            config=self,
+        )
+
+
+class SnowflakeConnectionConfig(BaseSettings):
```

Comment:
Maybe throw a comment/docstring about this somewhere explaining that it can be any ol' snowflake creds as long as it has write access. Probably most folks won't be as slow as me, but assumed I needed specific ones.

Reply:
added in [e3fda94](https://github.com/great-expectations/great_expectations/pull/10586/commits/e3fda942ac3729fa4403db37097ea3373c7c1207)